.gitlab-ci.yml

```diff
@@ -76,11 +76,11 @@ before_script:
     name: "$CI_JOB_NAME"
     paths:
       - _install_ci
+      # All those are for the test-suite jobs, to be discarded soon
       - config/Makefile
       - config/coq_config.py
       - config/coq_config.ml
       - config/dune.c_flags
-      - dmesg.txt
```

Comment:
this is still removed for no reason

Reply:
Has it ever been useful?

Reply:
nope, at least that I know of. But the other jobs still generate the file... either you don't generate it, or you also keep it

Reply:
I think it is kept, what job is generating it and not keeping it?

Reply:
You've removed the artefact from Gitlab here, so therefore it is not kept? I think you should revert this removal and leave the cleanup to the future.